### PR TITLE
2016-02-15 Content refresh

### PR DESCRIPTION
**Pull request by mdarby**
**References:**
https://github.com/getcarina/getcarina.com/issues/706
https://github.com/getcarina/getcarina.com/issues/692

### CODE REVIEW

File: _tutorials/2015-10-26-drupal-and-swarm.md [hunk 1]
```diff
@@ -30,6 +30,10 @@ This tutorial provides steps for running a Drupal instance and a MySQL database
 
     `source docker.env`
 
+1. Create an [overlay network](https://docs.docker.com/engine/userguide/networking/get-started-overlay/#step-3-create-the-overlay-network). This virtual network will privately encapsulate your containers for security.
```
Comment (everett-toews) [inline _tutorials/2015-10-26-drupal-and-swarm.md]: We typically don't link to anything in the body of a tutorial unless we expressly want the user to leave the tutorial and open that link. Here you already show the user how to create a network. There's no need to link out to other docs for it. Please remove the link.

File: _tutorials/2015-10-26-drupal-and-swarm.md [hunk 2]
```diff
@@ -30,6 +30,10 @@ This tutorial provides steps for running a Drupal instance and a MySQL database
...
+
+    `docker network create --driver overlay --subnet=10.0.9.0/24 mynetwork`
```
Comment (everett-toews) [inline _tutorials/2015-10-26-drupal-and-swarm.md]: Why did you choose this subnet?

File: _tutorials/2015-10-26-drupal-and-swarm.md [hunk 3]
```diff
@@ -37,11 +41,12 @@ This tutorial provides steps for running a Drupal instance and a MySQL database
   * `MYSQL_ROOT_PASSWORD` is the MySQL root username used to install the MySQL database.
 
       ```
-      $docker run --name mysql \
+      $ docker run --name mysql \
+      --net=mynetwork \
       -e MYSQL_USER=drupal \
-      -e MYSQL_PASSWORD=<password> \
+      -e MYSQL_PASSWORD='<password>' \
```
Comment (everett-toews) [inline _tutorials/2015-10-26-drupal-and-swarm.md]: Why include single quotes here and below? We don't do that anywhere else that I'm aware of.

**Comment by everett-toews** [comment]
@mdarby This merged a bit quickly. Can you please review the comments? It'll need a follow up PR.

File: _tutorials/2015-10-26-drupal-and-swarm.md (hunk 2 above)
Comment (mdarby) [inline _tutorials/2015-10-26-drupal-and-swarm.md]: was part of an example in the linked (about to be removed) Docker article.

File: _tutorials/2015-10-26-drupal-and-swarm.md (hunk 1 above)
Comment (mdarby) [inline _tutorials/2015-10-26-drupal-and-swarm.md]: The link was for more info. I will remove.

File: _tutorials/2015-10-26-drupal-and-swarm.md (hunk 3 above)
Comment (mdarby) [inline _tutorials/2015-10-26-drupal-and-swarm.md]: Unsure of everywhere else; I ran the commands as written and needed to wrap the info in some string.

**Pull request merged**